### ISSUE TITLE
Add hotfixes for i3-gaps-wm to allow configuring "tiling_drag"

### ISSUE THREAD

**Issue by davidgengenbach** [regolith-linux/regolith-desktop#787]
- [x] I've [run a search on the existing enhancements](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [x] I've read [the Regolith documentation](https://regolith-linux.org/docs/https://regolith-linux.org/docs/) and didn't find any relevant topic
- [] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Is your feature request related to a problem? Please describe.**
i3 added a feature where one can drag the title bar item to do splits via the mouse.

**Describe the solution you'd like**
Allow for the configuration of the `tiling_drag` option that was released as a bug fix release in https://github.com/i3/i3/issues/5209

https://build.i3wm.org/docs/userguide.html#config_tiling_drag

**Describe alternatives you've considered**
Installing `i3-gaps-wm` manually. I would only consider doing this when there is no immediate fix in sight.

**Additional context**

**Comment by davidgengenbach** [comment]
I saw that `i3-gaps-wm` will be integrated in the `i3` project! :partying_face: So this problem will solve itself over time.

My workaround for now (until either the hotfix is applied or i3 4.2.2 will be used in regolith) is:
```
sudo apt remove i3-gaps-wm --yes
sudo apt install i3 --yes
```

Strictly not important - and only an opinion:
The `tiling_drag` functionality was **really, really** annoying for me as I have muscle memory that lead to unintended tilings - all the heckin time.
I do not understand why the i3 developers added such functionality without making it configurable behind a flag in the first place and had to do a hotfix that in the case of regolith was not applied because of the `i3 -> i3-gaps-wm -> regolith/i3-gaps-wm` deployment chain.
That said: I am completely happy with i3/regolith and grateful to all the developers! :bowing_man:

**Comment by ristomatti** [comment]
@davidgengenbach I don't know if it's been just added but there seems to be a flag mentioned in the documentation https://i3wm.org/docs/userguide.html#config_tiling_drag?





> sudo apt remove i3-gaps-wm --yes

You migth want to remove the `--yes` part. At least I won't dare answer `Y` to this prompt without knowing what would happen if `regolith-desktop-complete` is removed with it?

```shell
$ sudo apt remove i3-gaps-wm
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libxcb-cursor0 regolith-rofi-config regolith-unclutter-xfixes
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED
  i3-gaps-wm i3-snapshot regolith-desktop-complete regolith-gnome-flashback regolith-i3-gaps-config
0 to upgrade, 0 to newly install, 5 to remove and 107 not to upgrade.
After this operation, 1,178 kB disk space will be freed.
Do you want to continue? [Y/n]  
```

**Comment by kgilmer** [comment]
@davidgengenbach , yes we'll get gaps features in the next i3 release I believe so I think that will be the real fix to your issue.

@ristomatti , judging from your terminal snippet I would guess you are on an older 1.x version of Regolith.  Newer versions added the ability to swap between i3 and i3-gaps easily.  This is not supported in 1.x and without some tinkering, it is not simple to use i3.  I would suggest upgrading to the latest Regolith version and using `i3` if you're impacted by the issue.

**Comment by ristomatti** [comment]
@kgilmer Correct. I had completely missed the new version....
